packages/generator/src/sdk-metadata/sdk-metadata.ts

```diff
@@ -33,6 +33,17 @@ export async function sdkMetaDataJS(
   };
 }
 
+export async function sdkMetaDataJSFallback(
```

Comment:
According to our styleguide it should be:
```suggestion
export async function sdkMetaDataJsFallback(
```